releasenotes/notes/add-hf-initial-point-ac29baf3091fa5b1.yaml

```diff
@@ -0,0 +1,51 @@
+---
+features:
+  - |
+    Adds Hartree-Fock (HF) and vibrational self-consistent field (VSCF) initial
+    point classes.
+    :class:`~qiskit_nature.initial_points.HFInitialPoint` and
+    :class:`~qiskit_nature.initial_points.VSCFInitialPoint`,
+    are to be used with :class:`~qiskit_nature.circuit.library.UCC`, and
+    :class:`~qiskit_nature.circuit.library.UVCC`, respectively.
+
+    This follows the introduction of
+    :class:`~qiskit_nature.initial_points.MP2InitialPoint` for
+    computing the initial point using the Møller-Plesset 2nd Order (MP2)
+    corrections.
+
+    :class:`~qiskit_nature.algorithms.initial_points.MP2InitialPoint` now
+    inherits from
+    :class:`~qiskit_nature.algorithms.initial_points.HFInitialPoint`, while
+    :class:`~qiskit_nature.algorithms.initial_points.HFInitialPoint` and
+    :class:`~qiskit_nature.initial_points.VSCFInitialPoint` both inherit
+    from the abstract base class
+    :class:`~qiskit_nature.initial_points.InitialPoint`.
+
+    These initial points are intended to be used as starting VQE parameters when
+    using a UCC ansatz (cluster operator coefficients). This should introduce an
+    initial point that is closer to the ground state point, leading to fewer
+    overall evaluations for VQE.
+
+    Instances of the appropriate classes are now the default
+    :attr:`initial_point` arguments in
```

Comment:
This fails to resolve since its not on one of the classes below so its just highlighted which maybe is ok/